app/assets/stylesheets/history.scss

```diff
@@ -0,0 +1,237 @@
+@import 'variables';
```

Comment:
This file is essentially `readinglist.css` with the names of the classes replaced and one additional class for the pagination button.